doc/radosgw/s3select.rst

```diff
@@ -261,25 +262,24 @@ s3-select function interfaces
 Timestamp functions
 ~~~~~~~~~~~~~~~~~~~
     | The timestamp functionalities as described in `AWS-specs <https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-glacier-select-sql-reference-date.html>`_  is fully implemented.
-    | the casting operator( ``to_timestamp( string )`` ), converts string to timestamp basic type.
-    | to_timestamp operator able to convert the following ``YYYY-MM-DD-HH-MI-SS.sssss(micro secs)+-HH:MI`` string format into timestamp
-    | upon time(or part of it) is missing, zero's are replacing the missing parts.
-    | +/- HH:MI is also optional, in the case it's missing, it's considere as 00:00. 
-    | values between -12:00 and 14:00 set the timezone of the timestamp value. 
 
-    | ``extract(date-part, timestamp)`` : function return integer according to date-part extract from input timestamp.
-    | supported date-part : year,month,week,day,hour,minute,second,timezone_hour,timezone_minute.
+    | ``to_timestamp( string )`` : The casting operator converts string to timestamp basic type.
+    | to_timestamp operator is able to convert the following ``YYYY-MM-DDTHH:mm:ss.SSSSSS+/-HH:mm`` , ``YYYY-MM-DDTHH:mm:ss.SSSSSSZ`` , ``YYYY-MM-DDTHH:mm:ss+/-HH:mm`` , ``YYYY-MM-DDTHH:mm:ssZ`` , ``YYYY-MM-DDTHH:mm+/-HH:mm`` , ``YYYY-MM-DDTHH:mmZ`` , ``YYYY-MM-DDT`` or ``YYYYT`` string formats into timestamp.
+    | Where time(or part of it) is missing in the string format, zero's are replacing the missing parts. And for missing month and day, 1 is default value for them.
+    | Timezone part is in format ``+/-HH:mm`` or ``Z`` , where the letter "Z" indicates Coordinated Universal Time (UTC). Value of timezone can range between -12:00 and +14:00.
 
-    | ``dateadd(date-part, integer, timestamp)`` : function returns timestamp, a calculation results of input timestamp and date-part.
-    | supported data-part : year,month,day,hour,minute,second.
-    | dateadd operator includes timezone in calculation.
+    | ``extract(date-part from timestamp)`` : The function extracts date-part from input timestamp and returns it as integer.
+    | Supported date-part : year, month, week, day, hour, minute, second, timezone_hour, timezone_minute.
 
-    | ``datediff(date-part, timestamp, timestamp)`` : function returns an integer, a calculated result for difference between 2 timestamps according to date-part.
-    | supported date-part : year,month,day,hour,minute,second.
-    | datediff operator includes timezone in calculation.
+    | ``date_add(date-part, quantity, timestamp)`` : The function adds quantity(integer) to date-part of timestamp and returns result as timestamp. It also includes timezone in calculation.
```

Comment:
:+1:

Reply:
space needed between quantity and (integer)?

Reply:
no, space is not needed (user can use spaces or not)

Reply:
this update move to #41834